src/main/java/de/muenchen/mobidam/s3/S3RouteBuilder.java

```diff
@@ -50,9 +61,19 @@ public void configure() {
                 .process(exchange -> {
                     if (exchange.getMessage().getBody()instanceof ErrorResponse res) {
                         exchange.getMessage().setHeader(Exchange.HTTP_RESPONSE_CODE, res.getStatus());
+                        if (exchange.getIn().getHeader(Constants.CAMEL_SERVLET_CONTEXT_PATH).equals(Constants.ARCHIVE_ENTITY)) {
+                            log.error(
+                                    String.format("Archive clean up (%s): %s", exchange.getProperty(Constants.ARCHIVE_ENTITY, MobidamArchive.class).toString(),
+                                            res.getError()));
+                        }
                     } else {
                         Throwable exception = exchange.getProperty(Exchange.EXCEPTION_CAUGHT, Throwable.class);
-                        log.error("Error occurred in route", exception);
+                        if (exchange.getIn().getHeader(Constants.CAMEL_SERVLET_CONTEXT_PATH).equals(Constants.ARCHIVE_ENTITY)) {
```

Comment:
Das kann man in eine Methode "logException" auslagern.

Reply:
✅